documentation/services_master_puppetserver.markdown

```diff
@@ -17,7 +17,7 @@ This page describes the generic requirements and run environment for Puppet Serv
 
 Puppet provides Puppet Server packages for Red Hat Enterprise Linux, RHEL-derived distros, Fedora, Debian, and Ubuntu.
 
-If we don't provide a package for your system, you can run Puppet Server from source on any POSIX server with JDK 1.7 or later. See [Running from Source](./dev_running_from_source.markdown) for more details.
+If we don't provide a package for your system, you can run Puppet Server from source on any x86_64 Linux server with JDK 1.8 or 11. See [Running from Source](./dev_running_from_source.markdown) for more details. Other POSIX compatible servers or versions of Java are unsupported by Puppet, although they may work. Join our [community Slack](https://slack.puppet.com/) from help with non-supported operating systems, architectures, or JRE versions.
```

Comment:
dangit... I must have missed it the first time, but that last line should say "for help" rather than "from help".

Reply:
Damn. Looks like you did it correctly in your suggestion. The first suggestion I just copy pasted, but the second one I read the suggestion and original sentences and tried to update accordingly. I don't think I would have _ever_ seen that.

Reply:
Okay, fixed for reals this time.